src/components/accession2/history/Accession2History.tsx

```diff
@@ -62,6 +62,11 @@ export default function Accession2History(props: Accession2HistoryProps): JSX.El
           </Typography>
           <Typography fontWeight={500}>
             {item.fullName || strings.NAME_UNKNOWN}&nbsp;{item.description}
+            {item.notes && (
+              <Typography fontSize={'14px'} fontWeight={300}>
```

Comment:
Not sure if setting the font size and weight to hardwired constants is the idiomatic way to do this; I was following the example of the parent `<Typography>` and `<Box>` tags.

Reply:
This works, we are using hardwired constants for now across the board. There is a plan to enhance all this with more size semantics. For now this is totally fine.